src/aws/osml/model_runner/tile_worker/tile_worker.py

```diff
@@ -155,7 +163,7 @@ def _refine_features(self, feature_collection, image_info: Dict, metrics: Metric
                     # Note that this property search will be deprecated and removed in a future release.
                     tiled_image_geometry = self.property_accessor.get_image_geometry(feature)
                     if tiled_image_bbox is None and tiled_image_geometry is None:
-                        logging.info(f"Feature may be using deprecated attributes: {feature}")
+                        logging.warning("Feature may be using deprecated attributes.")
```

Comment:
We may want this to be debug. I bet this is going to log a warning for every single feature created by some of the 3rd party models in use today.

Reply:
Ack - will update